dockertest/dockercmd_unittests.py

```diff
@@ -284,13 +282,13 @@ def test_no_fail_docker_cmd(self):
 
         docker_command = self.dockercmd.DockerCmd(self.fake_subtest,
                                                   'unittest_fail')
-        self.assertRaises(self.xceptions.DockerExecError,
+        self.assertRaises(self.output.xceptions.DockerExecError,
```

Comment:
...clarification:  What I mean is, a subtest should be assume a ``xceptions.DockerOutputError`` would be thrown, and the output module should throw a ``xceptions.DockerOutputError``.  So the unittest should also be checking for a ``xceptions.DockerOutputError``.